vignettes/occds.Rmd

```diff
@@ -389,8 +414,8 @@ adae <- tibble::tribble(
     7, "Alveolar proteinosis", NA_character_,  NA_integer_
 )
   
-adae <- derive_vars_query(dataset = adae , dataset_queries = queries)
-head(select(adae, USUBJID, ASTDTM, AETERM, AESEQ, AEDECOD, starts_with("SMQ05"), starts_with("CQ06")))
+adae_query <- derive_vars_query(dataset = adae1 , dataset_queries = queries)
+head(select(adae_query, USUBJID, ASTDTM, AETERM, AESEQ, AEDECOD, starts_with("SMQ05"), starts_with("CQ06")))
 ```
```

Comment:
I think we should use `dataset_vignette()` to display the datasets in the example. At the moment it is confusing that for the `queries` dataset `CQ01` and `SMQ02` are displayed but for the `adae_query` dataset `SMQ05` and `SMQ06`.

Reply:
Done.